test/test_helper.rb

```diff
@@ -0,0 +1,15 @@
+ENV["RAILS_ENV"] ||= "test"
```

Comment:
may be we can remove the whole test folder as we only use rspec.
Thanks!

Reply:
Removed the folder bhai. Thank you.